frontend/src/lang/en.ts

```diff
@@ -267,7 +267,9 @@ export const en = {
           guidelines: {
             li0: "Stay home when you are sick, except to get medical care.",
             li1:
-              "Self isolate for 10 full days after symptoms first appeared (or starting the day after you had your test, if you have no symptoms).",
+              "Stay home for 5 days." +
```

Comment:
missing a space after the periods
![image](https://user-images.githubusercontent.com/4952042/149361766-30fff1d3-2fc6-43d4-99df-a8f81a37d557.png)

Reply:
PDF:
![image](https://user-images.githubusercontent.com/4952042/149362201-596fbdee-d662-49cd-b09f-450b4bf34efc.png)

Reply:
Good catch, thank you! Updated:
![Screen Shot 2022-01-13 at 9 49 28 AM](https://user-images.githubusercontent.com/80282552/149382814-d2a84dac-547c-4406-afec-7b89e1994ae0.png)